graylog2-web-interface/src/components/search/MessageTablePaginator.jsx

```diff
@@ -83,7 +87,7 @@ const MessageTablePaginator = React.createClass({
       newPage = Number(page);
     }
 
-    SearchStore.page = newPage;
```

Comment:
We should remove the import on `SearchStore`, it seems unused since this change.

Reply:
✅